src/components/ebay-select/index.js

```diff
@@ -7,11 +7,11 @@ const processHtmlAttributes = require('../../common/html-attributes');
 const observer = require('../../common/property-observer');
 const template = require('./template.marko');
 
-const listboxOptionsClass = 'listbox__options';
-const listboxExpanderClass = 'listbox__control';
+const listboxOptionsClass = 'combobox__options';
```

Comment:
Do you want to remove all the old references to `listbox`? These could easily be changed like `listboxOptionsClass` to `optionsClass`. I also see there are some other leftover ones in the event names and elsewhere, which could be updated.

Reply:
This will all be done when we split this into `select` and `combobox`. It's all leftover from when it started out as `listbox`.

So, I feel we can leave it for now, as we will be working to get the native select version out later. I think that will be a v1 type of breaking change, too.

Reply:
IMO this is a good opportunity to clean it up, given that this PR removes the last practical usage of `listbox`. Doesn't make much sense to have the remaining references, even pending later cleanup. We're still in major version 0, so breaking changes shouldn't be an issue.

Reply:
Hmm, I feel we should rename as many of these internal variables now as we can to avoid confusion for others who might be working on this later.

Reply:
Okay. Let me get on that today.